### PR TITLE
[Optimize](Row store) allow to set row_store_page_size for tables, change default value to 16KB

### PR DESCRIPTION
**Pull request by lxr599**
## Proposed changes

In current row storage format, under different page sizes, the compression effect of zstd on the hits data set (70 GB) is as follows.
| no_compression| zstd (default, page size=4KB) |zstd ( page size=16KB)  |zstd ( page size=32KB)  |zstd ( page size=64KB)  |
|:--------:| :---------:|:--------:|:--------:|:--------:|
| 105.76 GB |33.38 GB |22.07 GB|21.83 GB|18.29 GB|

When the page size is 16KB, the optimization is the greatest (33.89%). When the page size is increased, the effect is not obvious.

### CODE REVIEW

**Review by eldenmoon — COMMENTED** [review]
LGTM

**Comment by eldenmoon** [comment]
run buildall

**Review by eldenmoon — DISMISSED** [review]
LGTM

**Comment by eldenmoon** [comment]
change row_store_page_size to table property

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java [hunk 1]
```diff
@@ -3463,6 +3463,10 @@ private static void addOlapTablePropertyInfo(OlapTable olapTable, StringBuilder
             sb.append(olapTable.getCompressionType()).append("\"");
         }
 
+        // row column page size
+        sb.append(",\n\"").append(PropertyAnalyzer.PROPERTIES_ROW_COLUMN_PAGE_SIZE).append("\" = \"");
```
Comment (eldenmoon) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java:3467]: only disable when row store enabled

**Comment by lxr599** [comment]
run buildall

File: regression-test/data/query_p0/system/test_query_sys_tables.out [hunk 2]
```diff
@@ -183,6 +183,13 @@ test_view
 -- !sql --
 
 -- !sql --
```
Comment (eldenmoon) [inline regression-test/data/query_p0/system/test_query_sys_tables.out:185]: add seperate case for this property and do not change other cases's output

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

**Review by eldenmoon — DISMISSED** [review]
LGTM

File: cloud/cmake/thirdparty.cmake [hunk 3]
```diff
@@ -59,7 +59,6 @@ add_thirdparty(protobuf)
 add_thirdparty(thrift)
 add_thirdparty(crypto)
 add_thirdparty(openssl LIBNAME "lib/libssl.a")
-add_thirdparty(jemalloc LIBNAME "lib/libjemalloc_doris.a")
```
Comment (xiaokang) [inline cloud/cmake/thirdparty.cmake:62]: why delete it?

File: fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java [hunk 4]
```diff
@@ -95,6 +95,10 @@ public class PropertyAnalyzer {
     public static final String PROPERTIES_TIMEOUT = "timeout";
     public static final String PROPERTIES_COMPRESSION = "compression";
 
+    // row column page size, default 16KB
+    public static final String PROPERTIES_ROW_COLUMN_PAGE_SIZE = "row_column_page_size";
```
Comment (xiaokang) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java:99]: row_store_page_size

**Review by xiaokang — COMMENTED** [review]
suggest name row_store_page_size

**Comment by eldenmoon** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run p0

**Review by eldenmoon — DISMISSED** [review]
lgtm

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run external

**Review by eldenmoon — APPROVED** [review]
LGTM

**Comment by eldenmoon** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by eldenmoon — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**